src/Microsoft.Azure.WebJobs.Extensions.ServiceBus/Listeners/ServiceBusListener.cs

```diff
@@ -38,6 +38,7 @@ internal sealed class ServiceBusListener : IListener, IScaleMonitorProvider
         private ClientEntity _clientEntity;
         private bool _disposed;
         private bool _started;
+        private bool _isStopping;

```

Comment:
can we use one flag for single and batch dispatch?

Reply:
This flag is different from _started, it's used to prevent calling of StopAsync while a stop is in progress.  In batch mode, the operation is quick and its ok to get called multiple times but in single dispatch mode, we should not call Unregister* while its in progress.  I can alternately add a lock and check _started flag which is a better way to handle concurrency.